Makefile

```
@@ -83,6 +83,14 @@ test-unit: envtest-setup
 update-csv: ensure-operator-sdk ensure-helm
 	bash -x hack/update-csv.sh
 
+verify-csv: CSV_VERSION:=9.9.9
+verify-csv: CURRENT_TIMESTAMP:=$(shell yq '.metadata.annotations.createdAt' deploy/cluster-manager/olm-catalog/latest/manifests/cluster-manager.clusterserviceversion.yaml)
```

checkmake (0.3.2)

[warning] Target "verify-csv" defined multiple times (lines 86 and 87).

(uniquetargets)